### DISCUSSION TITLE
related projects

### DISCUSSION THREAD

**Discussion by jhamman** [carbonplan/ndpyramid#155] [Ideas]
This issue is meant to document other, related projects.

**Comment by jhamman** [comment]
Xarray-multiscale: https://github.com/JaneliaSciComp/xarray-multiscale

**Comment by jhamman** [comment]
scikit-image: https://scikit-image.org/docs/dev/auto_examples/transform/plot_pyramid.html

**Comment by jhamman** [comment]
openmicroscopy multiscale: https://ngff.openmicroscopy.org/0.3/#multiscale-md

**Comment by forman** [comment]
xcube: https://github.com/dcs4cop/xcube/blob/master/docs/source/mldatasets.md
